runtime/common/src/constants.rs

```diff
@@ -10,6 +10,7 @@ use frame_support::{
 	PalletId,
 };
 
+// Duplicated in runtime/frequency/build.rs to keep build dependencies low
```

Comment:
I did try importing it, but having runtime common as a build dependency was having issues and it was way too much for a few constants.